cli/src/format.rs

```diff
@@ -176,8 +176,19 @@ impl Format for Vec<ProjectListEntry> {
             let _ = writeln!(writer, "{table}");
         } else {
             let table = format_table::<fn(&ProjectListEntry) -> String, _>(self, &[
+                ("Organization Name", |project| {
+                    match project.group_name.as_deref().unwrap_or("").split_once('/') {
+                        Some((org_name, _)) => {
+                            print::truncate(org_name, MAX_NAME_WIDTH).into_owned()
+                        },
+                        None => String::new(),
+                    }
+                }),
                 ("Group Name", |project| {
-                    let group_name = project.group_name.as_deref().unwrap_or("");
+                    let org_and_group = project.group_name.as_deref().unwrap_or("");
```

Comment:
I'm surprised clippy isn't complaining about not using `unwrap_or_default()`.

Reply:
It's just much shorter like this, `unwrap_or_default` was kind of a mistake anyway.